src/sql/workbench/contrib/notebook/test/common/utils.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export const invalidRelativePathRegex = /^\.\.+(?=\.\.)/g;
+
+export function replaceInvalidLinkPath(href: string): string {
+	href = href.replace(invalidRelativePathRegex, '..\\');
```

Comment:
This doesn't seem to handle more than 2 sets of .. currently, is that intended?

i.e. 

`......\\test.ipynb` is turned into `..\..\test.ipynb` instead of `..\..\..\test.ipynb` as I would expect

Reply:
Yeah instead of using the regex which ends up matching the entire case of all preceding ..'s. We can still use that for htmlMarkdownConvertor conditional to check, but for parsing and conversion, this seems much simpler and works for any 2 sets of ..'s.